pkg/nimble/netif/nimble_netif.c

```diff
@@ -75,11 +75,15 @@ static void _notify(int handle, nimble_netif_event_t event, uint8_t *addr)
     }
 }
 
-static void _netif_init(gnrc_netif_t *netif)
+static int _netif_init(gnrc_netif_t *netif)
 {
     (void)netif;
 
-    gnrc_netif_default_init(netif);
+    int res = gnrc_netif_default_init(netif);
+    if (res < 0) {
+        goto end;
```

Comment:
No need to jump around if `res` isn't used anywhere else 
```suggestion
        return res;
```